packages/server/lib/util/tests_utils.ts

```diff
@@ -9,6 +9,10 @@ export const flattenSuiteIntoRunnables = (suite, tests = [], hooks = []) => {
     )
   }
 
+  if (!suite || !suite.suites) {
```

Comment:
maybe explain why this is necessary and link to the github issue that this is fixing, this is a specific regression